### ISSUE TITLE
Minor spelling issue in dialog?

### ISSUE THREAD

**Issue by CowwolfeSoftware** [dojohnso/OctoPrint-PrintJobHistory#16]
<img width="606" alt="image" src="https://github.com/dojohnso/OctoPrint-PrintJobHistory/assets/4109629/f7d512f7-f3e5-4205-b6bd-2b112097a037">

Think it should say "Spool Vendor" 

P.S.
 (actually, Vendor is British English, Vender is Am. English).
Sorry for the nit-pick, I write code for a living and these things nag me too. :-)

**Comment by dojohnso** [comment]
Oh I know _exactly_ what you mean. 😅

Good catch. I'm cleaning things up like this when I see them. I'll be sure to fix this one up too. Thanks!

**Comment by dojohnso** [comment]
ok this is weird, and might just be a template cache thing. I saw the "Vendorss" in my local version, scoured the code and couldn't find any reason for it. I fixed a couple other minor things and rebuilt and reinstalled locally and it's fixed. So  I'm going to close this for now. Let me know if it doesn't go away after the next update though. (not sure yet when that will be)

![image](https://github.com/dojohnso/OctoPrint-PrintJobHistory/assets/294549/56d5cfeb-32e0-43da-9d3d-58f64474ecf1)
